### PR TITLE
Enrichments revamp

### PR DESCRIPTION
**Pull request by mikelgg93**
- Now, each enrichment has its own home 🏡.
- 🆕 New reference image mapper description, with examples.
- 🆕 New enrichments description page.

For more info 📝, see the commits.

### CODE REVIEW

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  >
+  </v-img>
+</div>
+
+
+Sometimes an object will occlude the feature/object of interest. The reference image mapper may not know this, so a false positive mapping could occur. 
+
+If you need to remove the falsely mapped data points, there are a few workarounds.
+
+1. Use Events and Sections to trim out false positive sections as in the above tip.
```
Comment (N-M-T) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:168]:
```suggestion
1. Use Events and Sections to trim out false positive sections.
```

**Review by N-M-T — CHANGES_REQUESTED** [review]

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
+---
+description: Powerful tool to map gaze data from the scene camera onto a reference image 
+permalink: /invisible/explainers/enrichments/reference-image-mapper
+---
+
+# Reference Image Mapper
+
+Reference Image Mapper is a powerful tool to automatically map gaze onto to features/objects in the environment.
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:8]:
```suggestion
Reference Image Mapper is a powerful tool to automatically map gaze onto features/objects in the environment.
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+<PhotoGrid :images="[
+  require(`../../../media/invisible/rim/heatmap-0.jpg`),
+  require(`../../../media/invisible/rim/heatmap-1.jpg`),
+  require(`../../../media/invisible/rim/heatmap-2.jpg`),
+  require(`../../../media/invisible/rim/heatmap-3.jpg`),
+]"/>
+
+Let's take a look at what the Reference Image and Scanning Recording look like to produce these heatmaps.
+### 1. Magazine reading
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:35]: This overview is very functional, but the spacing between image, video and text could be nicer. I think fine for now, but we should touch this again during the Neon docs update!

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+  </div>
+  <div class="col-mcontainer">
+    <h4>Scanning video</h4>
+    <Youtube src="0U4H-uOIHlw"/>
+    We take a longer scanning recording, about 2 min. The angles and distances cover what a person might see whilst walking past or standing in front of the building.
+  </div>
+</div>
+
+## Scanning best practices
+
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:118]:
```suggestion
Try to follow these best practices when recording a scanning video:
```
Bullet point lists starting directly after a title look a bit ugly IMO.

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+- Make the recording while holding the Pupil Invisible glasses in your hand rather than wearing it on your head.
+- Record the object of interest from all possible angles and from all distances a subject may look. More diversity is better. Collecting sufficiently diverse viewpoints or a large object, like a building, may take longer than capturing a small object like a magazine.
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:120]:
```suggestion
- Record the object of interest from all possible angles and from all distances a subject may view them from. More diversity is better. Collecting sufficiently diverse viewpoints for a large object, like a building, may require you to move accordingly large distances.
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+- Move the glasses slowly while recording to avoid motion blur.
+- Make sure to have good contrast and that your scene lighting during scanning is similar to that during mapping.
+
+What works and what doesn’t?
+- These examples are **feature rich**. Reference Image Mapper needs enough salient visual content to produce a successful mapping.
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:125]:
```suggestion
- The scene must be **feature rich** like in the examples above. There has to be enough salient visual content to produce a successful mapping.
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+- The scenes have relatively **static** features in the environment. If there is a lot of movement or the objects change in appearance, the mapping *can* fail.
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:126]:
```suggestion
- The scene needs to have relatively **static** features in the environment. If there is a lot of movement or the objects change in appearance, the mapping *can* fail.
```
Maybe this point needs some examples to illustrate? Not sure if people would understand what "objects changing in their appearance" would mean in practice 🤔

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+
+::: tip
+**Ready to go?**
+Why not try replicating the above examples? Or even try it with your own use-cases! If you haven’t already, we recommend you check out the [Cloud Getting Started Guide](/invisible/getting-started/analyse-recordings-in-pupil-cloud/#analyse-recordings-in-pupil-cloud), which covers the basics of working with enrichments.
+:::
+
+## Validate gaze mapping
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:133]:
```suggestion
## Validate the gaze mapping
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+    max-width=90%
+  >
+  </v-img>
+</div>
+
+To check if gaze has been mapped successfully, use the side-by-side view:
+1. Select a recording
+2. Select the Reference Image Mapper Enrichment
+3. Select the Scene / Reference Image View
+4. If you want to visualize and evaluate the 3D model generated (white dots), just turn on the Point Cloud toggle!
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:149]:
```suggestion
1. Select a recording.
2. Select the Reference Image Mapper Enrichment.
3. Select the Scene / Reference Image View.
4. If you want to visualize and evaluate the 3D model generated (white dots), just turn on the Point Cloud toggle!
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+
+Now when you play back the recording you can see where gaze is mapped to on your reference image for validation
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:151]:
```suggestion
Now when you play back the recording you can see where gaze is mapped to on your reference image for validation.
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+  <v-img
+    class="rounded" 
+    :src="require('../../../media/invisible/rim/basketball-occlusion.png')"
+    max-width=400px
+  >
+  </v-img>
+</div>
+
+
+Sometimes an object will occlude the feature/object of interest. The reference image mapper may not know this, so a false positive mapping could occur.
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:165]:
```suggestion
Sometimes an object will occlude the feature/object of interest. The reference image mapper may not recognize this, so a false positive mapping could occur. 
```

File: src/invisible/explainers/enrichments/reference-image-mapper.md [hunk 12]
```diff
@@ -0,0 +1,211 @@
...
+
+If you need to remove the falsely mapped data points, there are a few workarounds.
+
+1. Use Events and Sections to trim out false positive sections.
+2. Manually remove the affected data points in the reference image mapper export, by finding the timestamp and deleting the row in the .csv export.
+
+## Repetitions
+
+**Use Sections to map portions of your recordings**
+
```
Comment (marc-tonsen) [inline src/invisible/explainers/enrichments/reference-image-mapper.md:175]:
```suggestion
```
Given that this comes right after an H2 it seems repetitive to have another header.

**Review by marc-tonsen — COMMENTED** [review]
Very nice overall! Thanks for the work @mikelgg93! 👍 

I have iterated on some typos and formulations. I have also made a few comments just for your consideration, but I think we could merge already once the typos are fixed!

File: src/invisible/explainers/enrichments/reference-image-mapper.md (hunk 3 above)
Comment (mikelgg93) [reply src/invisible/explainers/enrichments/reference-image-mapper.md:35]: Agree. Perhaps adding an additional row below and tuning the margins and paddings on the captions

**Comment by mikelgg93** [comment]
note https://github.com/pupil-labs/pupil-docs/pull/551#discussion_r988842360 when merging.

**Review by marc-tonsen — APPROVED** [review]

**Pull request merged**